### DISCUSSION TITLE
Interactive not refreshing

### DISCUSSION THREAD

**Discussion by cojarbi** [ericaltendorf/plotman#755] [Q&A]
Ive been using Interactive with a lot of success under various scenarios. 

My latest machine build is a multiple hard drives running ubuntu server, same as another one I'm running just fewer discs.

When I load interactive it just stalls and then displays for me to input my ssh password either at the top or at the bottom of interactive. after putting in the password it shows the screen but there is no refresh.

![image](https://user-images.githubusercontent.com/16444438/121112250-27d6fc80-c7de-11eb-9a17-f11e1805ce70.png)

If i input the password multiple times then it refreshes for a bit to ask the password again.

Im not sure how this ownership got mixed here but sudo plotman interactive says "sudo: plotman: command not found"

**Comment by racergoodwin** [comment]
Have you generated and copied the ssh keys from the plotter to the farmer/node? There is a good tutorial here for 20.04, but it will work on other version of Ubuntu [https://linuxize.com/post/how-to-set-up-ssh-keys-on-ubuntu-20-04/](https://linuxize.com/post/how-to-set-up-ssh-keys-on-ubuntu-20-04/)
Also, you shouldn't run plotman as sudo
